modules/mod_base/support/z_list_of_ids_filter.erl

```diff
@@ -22,6 +22,8 @@
 -spec filter(integer() | [integer()], fun((integer()) -> boolean()), any()) -> boolean() | [integer()].
 filter(Id, Fun, _Context) when is_integer(Id) ->
     Fun(Id);
+filter(Id, Fun, _Context) when is_binary(Id) ->
+    Fun(Id);
```

Comment:
Added this `binary` clause to get the doc [example with an integer](http://zotonic.com/docs/latest/ref/filters/filter_is_a.html#apply-to-a-single-resource-id) working.